lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_error_message.rb

```diff
@@ -1,14 +1,22 @@
 module ErrorMessage
+  def self.binary
```

Comment:
Consider replacing the constants that use `binary` with methods that take in `binary` as an argument e.g.

```
self.binary_not_found(binary_type)
"Could not find the #{binary_type}. Make sure you set the #{binary_type} path parameter to point to your #{binary_type}"
end
```